--- original/ghidra_9.0/Ghidra/Processors/6502/data/languages/6502.ldefs	2019-02-28 13:00:12.000000000 -0800
+++ modified/ghidra_9.0/Ghidra/Processors/6502/data/languages/6502.ldefs	2019-03-07 23:33:35.000000000 -0800
@@ -9,6 +9,7 @@
             version="1.0"
             slafile="6502.sla"
             processorspec="6502.pspec"
+            manualindexfile="../manuals/6502.idx"
             id="6502:LE:16:default">
     <description>6502 Microcontroller Family</description>
     <compiler name="default" spec="6502.cspec" id="default"/>